méli-mélo/th-choose-canada/index.html

```diff
@@ -0,0 +1,40 @@
+---
+title: Choose Canada theme - Documentation
+dateModified: 2025-03-26
+description: "Documentation for the Choose Canada theme."
+lang: en
+---
+<div class="wb-prettify all-pre"></div>
+
+<p>Refer to the working example for a visual of the below code.</p>
+
+<section id="examples">
+	<h2>Working examples</h2>
+	<ul>
+		<li><a href="choose-canada.html">Choose Canada landing page</a></li>
+	</ul>
+</section>
+
+<section>
+	<h2>Theme elements</h2>
+
+	<h3>Choose canada banner</h3>
+	<pre><code>&lt;div class="bg-cover bg-center choose-canada-banner" data-bgimg-srcset="images/banner.jpg 1920w, https://wet-boew.github.io/vocab/wb/utilities#no-image 767w">
```

Comment:
To match the demo, this change is required

```suggestion
	<pre><code>&lt;div class="bg-cover bg-center choose-canada-banner" data-bgimg-srcset="images/banner.jpg 767w">
```